app/assets/javascripts/discourse/tests/integration/components/user-menu/likes-notifications-list-test.js

```diff
@@ -0,0 +1,46 @@
+import { module, test } from "qunit";
+import { setupRenderingTest } from "discourse/tests/helpers/component-test";
+import { render } from "@ember/test-helpers";
+import { cloneJSON } from "discourse-common/lib/object";
+import NotificationFixtures from "discourse/tests/fixtures/notification-fixtures";
+import { hbs } from "ember-cli-htmlbars";
+import pretender from "discourse/tests/helpers/create-pretender";
+
+function getNotificationsData() {
+  return cloneJSON(NotificationFixtures["/notifications"].notifications);
+}
+
+module(
+  "Integration | Component | user-menu | likes-notifications-list",
```

Comment:
I don't quite agree that the tests here as considered integration tests when we're only testing the component in isolation.

Reply:
Acceptance tests are heavier/slower than integrations and in my mind if I can get away with an "integration" test then that's what I do. Honestly I'm not even sure what "integration" means in our case, if you look here https://github.com/discourse/discourse/tree/main/app/assets/javascripts/discourse/tests/integration/components every file tests a single component and this is the pattern I'm following 🤷